styleguide.config.js

```diff
@@ -75,6 +78,9 @@ module.exports = {
     },
     devServer: {
       disableHostCheck: true,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
```

Comment:
This is a solution from here: https://github.com/styleguidist/react-styleguidist/issues/1298#issuecomment-469951138 

It looks like styleguidist will eventually work without this, but I think it's fine for now.